editor/src/core/commenting/comment-hooks.tsx

```diff
@@ -237,25 +238,21 @@ export function useAddMyselfToCollaborators() {
     [loginState],
   )
 
-  const maybeUpdateCollaborators = React.useCallback(() => {
-    if (!isLoggedIn(loginState) || projectId == null) {
-      return
-    }
-    void updateCollaborators(projectId)
-  }, [projectId, loginState])
-
   const collabs = useStorage((store) => store.collaborators)
 
   React.useEffect(() => {
     if (collabs != null) {
       addMyselfToCollaborators()
     }
-    maybeUpdateCollaborators()
-  }, [addMyselfToCollaborators, collabs, projectId, maybeUpdateCollaborators])
+  }, [addMyselfToCollaborators, collabs, projectId])
 }
 
 export function useCollaborators() {
-  return useStorage((store) => store.collaborators)
+  return useEditorState(
+    Substores.restOfEditor,
```

Comment:
I suggest to add a new Substore for multiplayer related stuff.
Or maybe use an atom for collaborators instead of the editor state (it seems to be quite independent from other parts of the state)

Reply:
@gbalint I added a multiplayer substate here [`fbb22ca` (#4946)](https://github.com/concrete-utopia/utopia/pull/4946/commits/fbb22ca6154824df4ed206709effd2c9c59a746f) . would that work for you or would you prefer the atom instead?